### DISCUSSION TITLE
Is there a reason why the documentation doesn't mention the support for importing from compressed (gzip) CSV files?

### DISCUSSION THREAD

**Discussion by emres** [duckdb/duckdb#1645] [Q&A] [answered]
Hello,

Is there a reason why [DuckDB documentation](https://duckdb.org/docs/) doesn't mention the support for importing from **compressed** (**gzip**) CSV files?

I've just accidentally realized that it has such support (seen some examples in some [Issues](https://github.com/duckdb/duckdb/issues)). I've also checked that it works using a simple example on my system, e.g.:

```
~/code ➜ file test.csv.gz
test.csv.gz: gzip compressed data, was "test.csv", last modified: Sat Apr 17 14:10:51 2021, from Unix, original size modulo 2^32 281
```

and then:

```
D CREATE TABLE ontime AS SELECT * FROM read_csv_auto('test.csv.gz');
D DESCRIBE ontime ;
┌───────┬───────────┬──────┬─────┬─────────┬───────┐
│ Field │   Type    │ Null │ Key │ Default │ Extra │
├───────┼───────────┼──────┼─────┼─────────┼───────┤
│ id    │ INTEGER   │ YES  │     │         │       │
│ tstz  │ TIMESTAMP │ YES  │     │         │       │
└───────┴───────────┴──────┴─────┴─────────┴───────┘
D SELECT * FROM ontime ;
┌────┬────────────────────────────┐
│ id │            tstz            │
├────┼────────────────────────────┤
│ 1  │ 2020-12-31 19:25:58.745232 │
│ 2  │ 2020-12-31 19:25:59.745232 │
│ 3  │ 2020-12-31 19:26:00        │
│ 4  │ 2020-12-31 19:26:00.000001 │
│ 5  │ 2020-12-31 19:26:01.000001 │
│ 6  │ 2020-12-31 19:26:02.000001 │
│ 7  │ 2020-12-31 19:26:02.000123 │
│ 8  │ 2020-12-31 19:26:02.001234 │
│ 9  │ 2020-12-31 14:00:00.000001 │
└────┴────────────────────────────┘
```

But I couldn't see anywhere in the documentation anything about this feature. Either I'm missing it, or there is indeed no mention of it. If it's the second case, then is there a specific reason? E.g. experimental/unstable feature, etc.?

**Comment by Mytherin** [comment, marked as answer]
There is no specific reason this is not mentioned, other than perhaps we didn't think to mention it. The feature has been there for a long time (>1 year?) and should be considered stable. Please feel free to submit a PR to the [website repo](https://github.com/duckdb/duckdb-web) to remedy this :)

**Reply by emres** [reply]
Great! Thanks! :) 

I'll try to send a pull request to the website repo as soon as I can.

**Reply by emres** [reply]
Created the pull request: https://github.com/duckdb/duckdb-web/pull/41

**Reply by hannes** [reply]
Merged your PR, thanks
